packages/flutter_localizations/lib/src/l10n/material_ps.arb

```diff
@@ -39,19 +39,19 @@
   "dialogLabel": "خبرې اترې",
   "alertDialogLabel": "خبرتیا",
   "searchFieldLabel": "لټون",
-  "reorderItemToStart": "TBD",
-  "reorderItemToEnd": "TBD",
-  "reorderItemUp": "TBD",
-  "reorderItemDown": "TBD",
-  "reorderItemLeft": "TBD",
-  "reorderItemRight": "TBD",
-  "expandedIconTapHint": "TBD",
-  "collapsedIconTapHint": "TBD",
-  "remainingTextFieldCharacterCountZero": "TBD",
-  "remainingTextFieldCharacterCountOne": "TBD",
-  "remainingTextFieldCharacterCountOther": "TBD",
-  "refreshIndicatorSemanticLabel": "TBD",
-  "moreButtonTooltip": "TBD",
+  "moreButtonTooltip": "More",
```

Comment:
Still need to regenerate the localized sources.

Reply:
These strings do not have localized sources. They were set to 'TBD' because that was the standard before when we added new strings that needed to be localized. Now, we just use the English source as a placeholder until the localized source is present (for Pashto, it will not be since Google does not support it).

Reply:
But I think he is saying that the `material_ps.arb` has been updated, so don't we need to run `gen_localizations` update the generated localization info with this changes to Pashto? Just like we would need to do if these updates came from the translation team.

Reply:
Right, that's what I was getting at.

Reply:
Oops, thanks for the clarification! Done